pkg/framework/aries/framework.go

```diff
@@ -76,6 +77,14 @@ func WithStoreProvider(prov storage.Provider) Option {
 	}
 }
 
+// WithProtocolSvc injects a protocol service to the Aries framework
+func WithProtocolSvc(new func(prv api.Provider) (api.ProtocolSvc, error)) Option {
```

Comment:
Suggestion: Variadic args? Just to avoid calling it again and again in case multiple services?

Reply:
done